tests/lax_scipy_test.py

```diff
@@ -190,6 +190,13 @@ def testLogSumExpNans(self):
         result = lsp_special.logsumexp(1.0, b=1.0)
         self.assertEqual(result, 1.0)
 
+  def testLogSumExpWhere(self):
+    x = jnp.array([[0., 1., 2.], [1., 2., 3.], [3., 4., 5.]])
+    mask = jnp.array([[True, False, True], [False, False, True], [False, True, True]])
+    y_expected = jnp.array([2.126928, 3., 5.313262])
```

Comment:
Rather than hard-coding, I'd suggest computing this reference value, for example using `scipy.special.logsumexp(x[mask])`. Otherwise it's unclear where these magic target values come from.

Reply:
Done.